packages/order-utils/test/signature_utils_test.ts

```diff
@@ -99,6 +99,17 @@ describe('Signature utils', () => {
             );
             expect(isValidSignatureLocal).to.be.true();
         });
+
```

Comment:
Maybe also add the test for a happy path?

Reply:
See other file.